packages/openapi/src/openapi-request-builder.ts

```diff
@@ -138,6 +141,16 @@ export class OpenApiRequestBuilder<ResponseT = any> {
     );
   }
 
+  /**
+   * Set the basePath that gets prefixed to the pathPattern before a request.
```

Comment:
[req] Similar. Don't use variable name directly. If people don't understand the context they will simply get lost.

Reply:
Again, I feel it is vague if I just say `Set the basePath that gets prefixed`, can I use {pathPattern} instead?

Reply:
[req] Also please mention (also in other places) that this is custom.

Reply:
Used API path pattern instead of the variable name.